framework/src/st_things/things_stack/cloud/cloud_connector.c

```diff
@@ -83,15 +80,9 @@ int CICheckDomain(const char *DomainName, char **pIP)
 		}
 	} 
 
-	THINGS_LOG_E(TAG, " Failed to get the IP, hard coding the ip");
-	snprintf(ipbuffer, sizeof(ipbuffer), "%s", "13.124.51.231");
-
-	usleep(500);
-	if (pIP != NULL) {
-		*pIP = things_strdup(ipbuffer);
-	}
+	THINGS_LOG_E(TAG, " Failed to get the IP");
 
-	return 0;
+	return -1;
```

Comment:
According to description ci_connection_pre_check, 
0 means success and > 0 means failure.
You can change return value or description.
I suggest description for this.

Reply:
Good. 
I will modify.
Thanks alot.

Reply:
@JeongJunSik  junsik, i have a question. 
if (pip == NULL)
 return 0; // is that correct~? i think should return the error value.

Reply:
yes, it is correct. 
there is below logic 
"if (ci_connection_pre_check(NULL, NULL) == 0) "
for checking internet connection.